<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EVSEWindow</class>
 <widget class="QWidget" name="EVSEWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>10</number>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="Interval">
          <property name="title">
           <string>Wiederholungsintervall</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QSpinBox" name="intevalSpinBox">
             <property name="suffix">
              <string>ms</string>
             </property>
             <property name="prefix">
              <string>Intervall: </string>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>19999</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="intervalButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Setzen</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="startLoading">
          <property name="title">
           <string>Ladung starten</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QSpinBox" name="startSpinBox">
             <property name="suffix">
              <string>A</string>
             </property>
             <property name="prefix">
              <string>Stromstärke:</string>
             </property>
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="startButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="digitalWrite">
          <property name="title">
           <string>Schreiben digitaler Daten</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="1">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QSpinBox" name="digitalSpinBoxPin">
             <property name="prefix">
              <string>Pin D</string>
             </property>
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>13</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="digitalButton1">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="digitalButton0">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="stopLoading">
          <property name="title">
           <string>Ladung beenden</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QPushButton" name="stopButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="PWM">
          <property name="title">
           <string>Pulsweitenmodulation 1kHz (Pin 10)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QSpinBox" name="pwmSpinBox">
             <property name="maximum">
              <number>249</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pwmButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDial" name="pwmDial">
             <property name="maximum">
              <number>249</number>
             </property>
             <property name="notchTarget">
              <double>24.899999999999999</double>
             </property>
             <property name="notchesVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="mainTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Schlüssel</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wert</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="stateTable">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashDotDotLine</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>60</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>/</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>A</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>B</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>C</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>D</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>E</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>F</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Code</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Beschreibung</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>/</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>Fehler!</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>Kein Fahrzeug angeschlossen</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>Fahrzeug angeschlossen</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>Ladevorgang ohne Lüfter</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>D</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string>Ladevorgang mit Lüfter</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>E</string>
           </property>
          </item>
          <item row="5" column="1">
           <property name="text">
            <string>Kurzschluss oder Spannungsausfall</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>F</string>
           </property>
          </item>
          <item row="6" column="1">
           <property name="text">
            <string>Tankstelle nicht verfügbar</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="forceIndicator">
          <property name="text">
           <string>Force: unbekannt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Senden</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTextBrowser" name="networkBrowser">
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pwmDial</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>pwmSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwmSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pwmDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
